content/en/news/2019/announcing-1.3.5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.3.5
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-11-06
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.5
+aliases:
+    - /news/announcing-1.3.5
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.4 and Istio 1.3.5.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** Envoy listener configuration for TCP headless services. ([Issue #17748](https://github.com/istio/istio/issues/17748))
+- **Fixed** an issue which caused stale endpoints to remain even when a deployment was scaled to 0 replicas. ([Issue #14436](https://github.com/istio/istio/issues/14336))
+- **Fixed** Pilot to gracefully handle generating invalid Envoy configuration. ([Issue 17266](https://github.com/istio/istio/issues/17266))
```

Comment:
this doesn't make alot of sense to me.  Is Pilot generating invalid envoy configuration and that is what was fixed?

Reply:
If pilot does generate invalid config, previously it would crash. Now it doesn't. This makes us resiliant to bugs that we don't know about, and prevents https://istio.io/docs/reference/config/networking/v1alpha3/envoy-filter/ from crashing pilot

Reply:
How about:
```
- **Fixed** Pilot to gracefully handle Envoy rejecting configurations. (...)
```

Reply:
- **Fixed** Pilot to no longer crash when an invalid Envoy configuration is generated.